src/main/java/com/example/zawody/model/Result.java

```diff
@@ -0,0 +1,45 @@
+package com.example.zawody.model;
```

Comment:
`zawody` is no so well package name. You could also rename the base package to `com.gmail.youremailname`

Reply:
done